ena_upload/templates/ENA_template_studies.xml

```diff
@@ -8,16 +16,29 @@
         <DESCRIPTOR>
             <STUDY_TITLE>${row.title}</STUDY_TITLE>
             <STUDY_TYPE existing_study_type="${row.study_type}"/>
+            <py:if test="attributetest(row, 'study_abstract')">
```

Comment:
In SRA.study.xsd the abstract is not mandatory, but in the webin submission portal it is.
Might be worth testing if ENA is accepting programmatic submission of studies without an abstract.

Reply:
@rabuono would you like to do that? :D

Reply:
With an empty `study_abstract` but with `pubmed_id` with values it works.
Will try without both.

Reply:
With both `study_abstract` and `pubmed_id` without values, a submission to their dev/test service was successful.
However, since they make the `study_abstract` a mandatory field in their webin submissions portal, it might be best to also keep it as a mandatory field here

Reply:
ok made it mandatory again